src/chrome/content/rules/itcafe.hu.xml

```diff
@@ -0,0 +1,5 @@
+<ruleset name="itcafe.hu">
+	<target host="itcafe.hu" />
```

Comment:
Please also add:
`www.itcafe.hu` (mismatch, redirect to itcafe.hu)
`m.itcafe.hu`